.circleci/config.yml

```diff
@@ -28,35 +59,50 @@ jobs:
       - codecov/upload:
           file: lightning-lsp-common/coverage/*.json
           flags: frontend,common
-      - save_cache:
+
+  test-on-windows:
+    description: "Run tests on windows"
+    executor: win/default
+    steps:
+      - checkout
+      - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
-          paths:
-            - node_modules
+      - run: yarn install
+      - run:
+          name: Setup workspace Symlinks for Windows
+          command: yarn windowsCopySymlinks
+      - run:
+          name: Run tests on windows (Without coverage)
+          command: yarn test
 
-  # TODO: Don't recompile everything. Use workspace persistance.
+  # Publish to NPM
   deploy:
     <<: *defaults
     description: "Publish package to npmjs.org"
     steps:
-      - checkout
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
+      - attach_workspace:
+          at: ~/lightning-language-server
       - run: yarn install
       - run: yarn build
-      - run: git config user.email "nkruk@salesforce.com"
-      - run: git config user.name "Nick Kruk"
+      - run: git config user.email "kgray@salesforce.com"
+      - run: git config user.name "Kris Gray"
+      - run: npm config set '//registry.npmjs.org/:_authToken' "${NPM_PUBLISH_TOKEN}"
```

Comment:
Doing this to avoid having it in the NPMRC file. 

That is problematic for 2 reasons.

If you just put the Environment variable name in there, building fails unless you declare that value on the build host.
If I don't check that in, and do it at deploy time, lerna complains we have a changed file when its trying to do some stuff around pushing versions back to git.

Will verify this once it gets back to master.